src/telegramWebHook.js

```diff
@@ -64,7 +64,7 @@ class TelegramBotWebHook {
     debug('WebHook request headers: %j', req.headers);
 
     // If there isn't token on URL
-    if (!this.regex.test(req.url)) {
+    if (!this.urlValidator(req.url)) {
```

Comment:
Why don't you call `indexOf()` here, so that it be:

```js
if (req.url.indexOf(this.token) === -1) {
```

Seems simpler and avoids the func call (although the code may be automatically optimized for that, by v8).